### DISCUSSION TITLE
Two standalone minio on two namespaces in the same k8s cluster

### DISCUSSION THREAD

**Discussion by f-lalonde** [minio/minio#18361] [Q&A] [answered]
Good day, all!
I'm having some issues, so before persevering, I just want to make sure that I didn't miss anything and that what I'm looking to do isn't impossible for some reason.

I am upgrading all Minios on our services. Usually, there's only one Minio per cluster, but an internal test server has two of them. I am running the first one on ports 9000/9001, and am trying to set up the new one on 9002/9003. 

All seem to go great, it runs, but when I try to register the new one as an alias with MC, I get the following error : 
`mc: <ERROR> Unable to initialize new alias from the provided credentials. Get "http://minio-staging.myhost.com:9002/probe-bucket-sign-6amos05kpmr0/?location=": Connection closed by foreign host http://minio-staging.myhost.com:9002/probe-bucket-sign-6amos05kpmr0/?location=. Retry again.
`

I can work around that by simply setting different buckets instead of different clients (they are stored on the same NFS drive, though different folders). However, I would like to know if its possible, and if it is, I'd like to try and debug it, just in case we ever need to do this in production for unforeseen reasons.

Thanks!

**Comment by harshavardhana** [comment, marked as answer]
NFS volumes are terrible please don't use them for MinIO. Object store over NAS is a big no no if you are on a distributed setup.

Please use regular disks with XFS on top.

**Reply by f-lalonde** [reply]
Nice, hiring those contractors really was worth it. Thanks for your answer, I'll try and connect our Minio straight to an S3 service with our cloud storage services instead.
